README.md

```diff
@@ -1,2 +1,109 @@
 # Avo Inspector SDK for Java
 
+[![](https://jitpack.io/v/avohq/java-avo-inspector.svg)](https://jitpack.io/#avohq/java-avo-inspector)
+
+# Avo documentation
+
+This is a quick start guide.
+For more information about the Inspector project please read [Avo documentation](https://www.avo.app/docs/implementation/inspector/sdk/java).
+
+# Installation
+
+We host the library on JitPack.io, so
+
+add the following to the build.gradle file:
+
+```
+    repositories {
+      ...
+      maven { url 'https://jitpack.io' }
+    }
+```
+
+and:
+
+```
+    dependencies {
+        implementation 'com.github.avohq.java-avo-inspector:TAG'
+    }
+```
+
+Use the latest GitHub release tag to get the latest version of the library.
+
+# Initialization
+
+Obtain the API key at [Avo.app](https://www.avo.app/welcome)
+
+```java
+import is.avo.inspector.AvoInspector;
+import is.avo.inspector.AvoInspectorEnv;
+
+AvoInspector avoInspector = new AvoInspector("MY_API_KEY", "1.0.0", "My App Name", AvoInspectorEnv.Dev);
+```
+
+# Enabling logs
+
+Logs are enabled by default in the dev mode and disabled in prod mode based on the init flag.
+
+```java
+AvoInspector.enableLogging(true);
+```
+
+# Sending event schemas
+
+Whenever you send tracking event call one of the following methods.
+
+Example usage:
+
+```java
+void trackAppOpened(Map<String, ?> appOpenedEventParams) {
```

Comment:
I have a feeling most non-avoCodegen users have this as 

`void trackEvent(string eventName, Map<String, ?> eventParams)`

Should we make the example reflect that ?

Reply:
IM sorry, i see that i copied a trackAppOpened method as the last comment. I got that somewhere from our docs 👀 
I feel like both scnearios is better with trackEvent though right ?


If i were to make a simple wrapper around sending events to mixpanel and X, i'd never make something with function per event

Reply:
True, but I feel like single event example more verbose and easy enough to extrapolate to the wrapper function